changelog.md

```diff
@@ -1,3 +1,13 @@
+# v0.0.35
+
+## Breaking Changes
+
+## Updates
+
+## Development
+
+- Added Outlined TextField
```

Comment:
This should be moved under ##Updates and provide some more documentation on the TextField. Props etc...
```md
## Updates
### TextField
- created text field component outlined variant
- accepts ...
- accepts icons...
```